arelle/CntlrCmdLine.py

```diff
@@ -750,7 +750,7 @@ def monitorParentProcess():
             self.modelManager.disclosureSystem.select(None) # just load ordinary mappings
             self.modelManager.validateDisclosureSystem = False
         if options.utrUrl:  # override disclosureSystem utrUrl
-            self.modelManager.disclosureSystem.utrUrl = options.utrUrl
+            self.modelManager.disclosureSystem.utrUrl = [options.utrUrl]
```

Comment:
Why this change?

Reply:
DisclosureSystem expects utrUrl to be a list.  Looks like https://github.com/Arelle/Arelle/commit/e15b0a6c92f0ed9717575ce2bcb342dd03fa13e1 made it a list and missed this option.

Reply:
OK that makes sense because we added a FERC specific utr check since that taxonomy was using units that were not fully accepted into the UTR.